crates/autopilot/src/database.rs

```diff
@@ -50,6 +57,11 @@ async fn count_rows_in_table(ex: &mut PgConnection, table: &str) -> sqlx::Result
     sqlx::query_scalar(&query).fetch_one(ex).await
 }
 
+async fn count_rough_rows_in_table(ex: &mut PgConnection, table: &str) -> sqlx::Result<i64> {
```

Comment:
nit: Nice TIL about fetching estimated table sizes.
Just the function name seems a bit weird. Also maybe a small comment would be nice explaining that actually counting rows can be too slow for very big tables.
```suggestion
async fn estimate_rows_in_table(ex: &mut PgConnection, table: &str) -> sqlx::Result<i64> {
```